### PR TITLE
docs: Update TPP playbook

### PR DESCRIPTION
**Pull request by evansd**
There's a further piece of work here which is separating out the
TPP-specific stuff from the generic cross-backend stuff, but one thing
at a time.

### CODE REVIEW

File: playbooks/tpp-backend.md [hunk 1]
```diff
@@ -59,19 +100,52 @@ launch Windows Event Viewer directly pointing at the nssm events.
...
 
 The main log file is:
 
     /e/job-runner/service.err.log
 
 We also log stdout to `/e/job-runner/service.log` but that is usually
-empty.
+empty. Both these files are rotated by nssm.
+
+Use `tail -f` to see what's happening right not:
```
Comment (rebkwok) [inline playbooks/tpp-backend.md:121]: typo "now"

File: playbooks/tpp-backend.md [hunk 2]
```diff
@@ -59,19 +100,52 @@ launch Windows Event Viewer directly pointing at the nssm events.
...
 
-These files are rotated by nssm.
+    tail -f service.err.log
 
+Lines prefixed `sync` are from the synchronisation loop which
+communicates with the job's website i.e. polls for new jobs to do and
```
Comment (rebkwok) [inline playbooks/tpp-backend.md:126]: "jobs website" (no apostrophe - I think)

File: playbooks/tpp-backend.md [hunk 3]
```diff
@@ -81,64 +155,118 @@ the `.env` file:
...
 
+Look for the field labelled "Last Updated by Runner". This tells you
+when the job-runner last "checked in" with the job. Usually this is
+under a minute ago. If the timestamp is recent then there's not an issue
+with job-runner and you can proceed to step 2.
 
-Then restart the service
+If it's longer than a few minutes then it could just be that job-runner
+is busy copying some large files around. For simplicity's sake
+job-runner is single threaded and so if it's busy copy files for another
```
Comment (rebkwok) [inline playbooks/tpp-backend.md:181]: copy -> copying

File: playbooks/tpp-backend.md [hunk 4]
```diff
@@ -81,64 +155,118 @@ the `.env` file:
...
+timestamp still isn't updated then something has gone wrong. It's worth
+trying a restart of the job-runner but if that doesn't unstick it then
+you'll need to escalate.
 
 
-### Generic operations
+#### 2. Check the individual job log
 
-#### Update docker image
+Everything a job writes to stdout or stderr ends up in it's log file.
```
Comment (rebkwok) [inline playbooks/tpp-backend.md:199]: it's -> its

**Review by rebkwok — APPROVED** [review]
This is great, thanks! Just some pedantic comments on typos

**Review by ghickman — APPROVED** [review]
Great set of improvements!

You've expressed that various commands allow partial Job IDs, but I think it might be useful to add a small bit in the fundamentals section explaining that the inability to copy/paste is why that exists to add some context.

**Comment by evansd** [comment]
Thanks @rebkwok and @ghickman, I've updated based on your comments.

**Pull request merged**